testsuite/tests/kuadrant/identical_hostnames/test_identical_hostnames_rlp_on_gw_and_route.py

```diff
@@ -22,7 +22,7 @@ def rate_limit2(request, gateway, blame, cluster, label):
     return rlp
 
 
-def test_identical_hostnames_rlp_on_gw_and_route_ignored(client, rate_limit, rate_limit2, hostname):
```

Comment:
Could you extract these unrelated changes to a separate PR?

Reply:
Sure, will do

Reply:
Done, removed from this PR. Created https://github.com/Kuadrant/testsuite/pull/479 instead